README.md

```diff
@@ -431,6 +431,8 @@ For information about the governance of the Node.js project, see
 **Yosuke Furukawa** &lt;yosuke.furukawa@gmail.com&gt;
 * [ZYSzys](https://github.com/ZYSzys) -
 **Yongsheng Zhang** &lt;zyszys98@gmail.com&gt; (he/him)
+* [baradm100](https://github.com/baradm100) -
+**Bar Admoni** &lt;admoni.bar1@gmail.com&gt; (he/him)
```

Comment:
Contributors are not collaborators. There are over 3000 contributors to Node.js. We (obviously) don't list them all in the README.md.

Reply:
Sounds reasonable, my mistake, fixing now

Reply:
Fixed, resolving the conversation